charts/prometheus/templates/server/sts.yaml

```diff
@@ -110,21 +110,32 @@ spec:
           {{- if .Values.server.baseURL }}
             - --web.external-url={{ .Values.server.baseURL }}
           {{- end }}
+          {{- if .Values.server.listenOnLocalhost }}
+            - --web.listen-address=localhost:9090
```

Comment:
Imho if we make use of "--web.listen-address" it should be configurable and not bound to localhost only.